src/main/kotlin/no/nav/syfo/application/IVurderingRepository.kt

```diff
@@ -8,6 +8,10 @@ interface IVurderingRepository {
         personident: PersonIdent,
     ): List<Vurdering>
 
+    fun getVurderingForPersoner(
```

Comment:
Var ikke helt åpenbart i endepunktet at det er siste vurdering for hver person vi henter, kanskje repository og service-funksjonene burde hete noe ala `getLatestVurderingForPersoner`?